listenbrainz/bigquery_writer/bigquery_writer.py

```diff
@@ -69,17 +67,15 @@ def submit_data(self):
         while True:
             try:
                 ret = self.bigquery.tabledata().insertAll(
-                    projectId=self.config.BIGQUERY_PROJECT_ID,
-                    datasetId=self.config.BIGQUERY_DATASET_ID,
-                    tableId=self.config.BIGQUERY_TABLE_ID,
+                    projectId=current_app.config['BIGQUERY_PROJECT_ID'],
+                    datasetId=current_app.config['BIGQUERY_DATASET_ID'],
+                    tableId=current_app.config['BIGQUERY_TABLE_ID'],
                     body=body).execute(num_retries=5)
                 break
             except HttpError as e:
-                self.log.error("Submit to BigQuery failed: %s. Retrying in 3 seconds." % str(e))
+                current_app.logger.error("Submit to BigQuery failed: %s. Retrying in 3 seconds. Data: %s", str(e), json.dumps(body, indent=3), exc_info=True)
```

Comment:
should we be using ujson here since it was imported?

Reply:
`json.dumps` has an `indent=3` option which leads to more readable json, that's why i used it.